packages/alfa-dom/src/types.ts

```diff
@@ -432,3 +432,10 @@ export const enum Namespace {
    */
   XMLNS = "http://www.w3.org/2000/xmlns/"
 }
+
+/**
+ * Map of namespace alias (key) and namespace URI (value).
+ *
+ * @see https://www.w3.org/TR/selectors/#type-nmsp
+ */
+export type NamespaceDeclarations = Map<string | null, Namespace>;
```

Comment:
The namespace map is only ever used internally in the package so you can leave out the type alias.